### DISCUSSION TITLE
How to set the epoch in detectron2?? I can't find it

### DISCUSSION THREAD

**Discussion by leesangjoon1** [facebookresearch/detectron2#3622] [Q&A]
I wonder how to set the epoch in detectron2 ?? 

I set the batch and iteration but I can't find epoch

where is it?? 

Thanks!

**Comment by ppwwyyxx** [comment]
There is no concept of epoch in detectron2.

**Reply by leesangjoon1** [reply]
> There is no concept of epoch in detectron2.

Then How the deep learning training process is running??

for example, if the batch is 32, iteration is 100,000, the number of images is 1000, how the process is going? 

1,000 images are divided per 32 and go through to filter 100,000 times?

**Reply by haiduo** [reply]
You can compute the epoch  with equal to "  (max_iiteration(100,000) * batch_szie(32)) /  images_count(1000) "
